extensions/eol-runtime-scanner/www/styles.css

```diff
@@ -13,6 +13,77 @@
   margin-top: -10px;
 }
 
+/* Version selector styles */
+.version-selector {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 4px;
+  padding-top: 5px;
+}
+
+/* ActionButton version */
+.btn-version {
+  padding: 2px 6px;
+  font-size: 0.8rem;
+  font-family: "Fira Mono", Consolas, Monaco, monospace;
+  border: 1px solid #ddd;
+  background: #f8f9fa;
+  border-radius: 3px;
+  margin-right: 0;
+  margin-bottom: 4px;
```

Comment:
I'd remove the `margin-bottom` and use `gap` in `.version-selector` instead. If you want a gap in between the rows you can use `row-gap: 8px` and `column-gap: 4px` to achieve the same look.

That way you aren't using two methods to setup the spacing.